roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -19,15 +19,23 @@ class OpenShiftCheckException(Exception):
 
 @six.add_metaclass(ABCMeta)
 class OpenShiftCheck(object):
-    """A base class for defining checks for an OpenShift cluster environment."""
+    """
+    A base class for defining checks for an OpenShift cluster environment.
+
+    Expect a method execute_module, optional dict task_vars, and optional string tmp.
+    execute_module is expected to have a signature compatible with _execute_module
+    from ansible plugins/action/__init__.py, e.g.:
+    def execute_module(module_name=None, module_args=None, tmp=None, task_vars=None, **kwargs)
+    This is stored so that it can be invoked in subclasses as check.execute_module("name", args)
+    which provides the check's task_vars and tmp by default.
```

Comment:
If you're wondering why all the explanation here, it's for future me because I keep forgetting what the method signature should be between plugin `_execute_module` and check `execute_module`.